### PR TITLE
Add CMS design system grid

### LINKED ISSUE #1246: Add CMS design system grid to our project
The CMS design system separated their grid into an NPM package that's installable separately from the main system. I think it makes sense to install the grid first, since we already know from #1054 that our current grid isn't fulfilling all of our needs. It will also buy us a little time to figure out how the more visible elements of the design system affect what we've currently got going on.

### This task is done when...
 - [x] The grid package is installed
 - [ ] The pages have been updated to use the grid styles

### PR DESCRIPTION
**Pull request by hbillings**
Closes #1246, #1038.

Out with the old grid, in with the new grid!

### This pull request changes...
- the grid system
- left nav should now be a fixed width thanks to the new grid

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### This feature is done when...
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   db:
-    image: postgres:latest
+    image: postgres:10
```
Comment (mgwalker) [inline docker-compose.yml]: Thank you!

**Comment by hbillings** [comment]
@mgwalker I realized pairing this morning with Jen that the site's original grid was 10-column based, while the design system grid is 12-column based. So you're right -- proportionally, the main content was missing two columns! Good eye!

The loss of the sticky sidebar is definitely unintentional, and I'm not sure what I did to cause that. I'll poke around a bit more in this PR. If it seems like it's a big lift, I'll roll that work into the nav restyling issue I'm picking up next.

**Comment by hbillings** [comment]
Okay, the sticky sidebar was previously created by using `position: fixed` on the sidebar container. This worked because we were specifying the sidebar's `max-width` columns in the CSS. With the new grid system, the column width is specified as a class on the parent container, and setting `position: fixed` causes the children to overflow the container. I'm not entirely sure what the right fix for this is, but since we're going to need to restructure the nav to implement the new styles, I think it makes sense to bundle this with that. Are you good with that, @mgwalker & @jenniferthibault?

screen shot 2019-02-14 at 3 47 03 pm

**Comment by jenniferthibault** [comment]
Taking that on as part of #1267 works for me!

**Comment by mgwalker** [comment]
To make sure I understand, the plan is to merge it as-is and fix the sticky piece of the nav in #1267?  If so, that makes a ton of sense to me.  I'd ping in @jeromeleecms and @NAretakis for their awareness and feedback, though, if any.

**Comment by hbillings** [comment]
Thanks, both! I added a checkbox to the description of #1267 to make sure we don't lose that acceptance criterion. I'll ping the POs just to make sure.

**Pull request merged**